wallet/src/actors/app/handlers/create_data_req.rs

```diff
@@ -3,21 +3,107 @@ use serde::{Deserialize, Serialize};
 
 use crate::actors::app;
 use crate::types;
+use crate::types::{Hashable as _, ProtobufConvert as _};
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct CreateDataReqRequest {
-    pub rad_request: types::RADRequest,
+    session_id: types::SessionId,
+    wallet_id: String,
+    label: Option<String>,
+    request: DataRequestOutput,
+}
```

Comment:
We also need a `data_request_fee`, which is the fee for including the `DataRequestTransaction` into a block. The cost for creating a data request is `data_request.value + fee`.

```suggestion
    fee: u64
}
```